docs/deep-dives/mobile/behavioral-targeting.mdx

```diff
@@ -0,0 +1,156 @@
+
+| Name | Description | Targeting String |
+|:---|:---|:---|
+| Core Active Users | A user who has used the application 21 out of the last 28 days. | <pre><code>'app-opened-event'&#124;eventCountNonZero('Days', 28, 0) >= 21</code></pre> |
+| Recently Logged In Users | A user who has logged into Sync within the last 12 weeks. | <pre><code>'sync-signin-event'&#124;eventCountNonZero('Weeks', 12, 0) > 0</code></pre> |
+
+
+## Engineering
+
+In order to instrument a new behavior/event, an equivalent call to the Nimbus event recording method must be made alongside the call to record a Glean event.
+
+<Tabs
+  defaultValue="ios"
+  values={[
+    { label: "iOS", value: "ios" },
+    { label: "Android", value: "android" },
+  ]
+}>
+<TabItem value="ios">
+
+On Firefox for iOS, there is a wrapper for telemetry which makes this process rather simple. In the switch case for the event you want to implement, add a call to `Experiments.shared.recordEvent`. The argument should be the event name.
+
+```swift
+// Client/Telemetry/TelemetryWrapper.swift
+extension TelemetryWrapper {
+    // ...
+    static func gleanRecordEvent(category: EventCategory, method: EventMethod, object: EventObject, value: EventValue? = nil, extras: [String: Any]? = nil) {
+        switch (category, method, object, value, extras) {
+        // ...
+        case (.firefoxAccount, .view, .fxaLoginCompleteWebpage, _, _):
+            GleanMetrics.Sync.loginCompletedView.record()
+            // record the same event for Nimbus' internal event store
+            Experiments.shared.recordEvent("sync.login_completed_view")
+        // ...
+```
+
+[Example in PR](https://github.com/mozilla-mobile/firefox-ios/pull/12560)
+
+</TabItem>
+<TabItem value="android">
+
+On Firefox for Android, a call should be made to `components.analytics.experiments.recordEvent` immediately following the Glean event being recorded. The argument should be the event name.
+
+```kotlin
+// app/src/main/java/org/mozilla/fenix/HomeActivity.kt
+
+import org.mozilla.fenix.ext.components
+
+// ...
+open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
+    // ...
+    final override fun onCreate(savedInstanceState: Bundle?) {
+        // ...
+        if (settings().isTelemetryEnabled) {
+            // ...
+            safeIntent
+                ?.let(::getIntentSource)
+                ?.also {
+                    Events.appOpened.record(Events.AppOpenedExtra(it))
+                    // This will record an event in Nimbus' internal event store. Used for behavioral targeting
+                    components.analytics.experiments.recordEvent("app_opened")
+                }
+        }
+        // ...
+    }
+}
+```
+
+[Example in PR](https://github.com/mozilla-mobile/fenix/pull/27934)
+
+</TabItem>
+</Tabs>
+
+<hr/>
+
+The following are the events that are currently instrumented in Firefox for iOS and Android, respectively:
```

Comment:
Nit: I'd put this above the engineering section, and give it a heading— so we can link to it.